yang-packages/draft-ietf-netmod-yang-packages.txt

```diff
@@ -1226,32 +1153,32 @@ Internet-Draft                YANG Packages                 October 2022
       */
 
      typedef pkg-name {
-
-
-
-Wilton, et al.            Expires 28 April 2023                [Page 22]
-
-Internet-Draft                YANG Packages                 October 2022
-
-
        type yang:yang-identifier;
        description
          "Package names are typed as YANG identifiers.";
      }
 
      typedef pkg-version {
-       type rev:revision-date-or-label;
+       type ys:version;
        description
-         "Package versions SHOULD be a revision-label (e.g. perhaps a
-          YANG Semver version string).  Package versions MAY also be a
-          revision-date";
-
+         "Packages are versioning used YANG Semver version labels.";
      }
 
-     typedef pkg-identifier {
-       type rev:name-revision;
+     typedef version-or-rev-date {
```

Comment:
Ideally this should be in YANG semver document?

Reply:
Maybe, although, I'm not sure whether it really matters.  Potentially having it in the YANG Semver would mean that it is in only one place. Thoughts @xorrkaz ?